mergestat/mergestat
# feat(import):Managing auto import errors

here is our first iteration for managing impor errors in the worker .
To test this implementation please do:

1. make docker-clean
2. make dev
3. loging into the mergestat
4. add credentials
5. add an automatic import with a unexisting organization or user
6. check the logs  for warn type msg   to see the error ocurred
7. use the ID given in the logs to search that specific import failure 
8. :)
<img width="1440" alt="Screen Shot 2022-11-21 at 2 26 01 PM" src="https://user-images.githubusercontent.com/43893061/203143602-1aeae0e5-390f-45b4-a97c-6b6083ba25aa.png">
<img width="1440" alt="Screen Shot 2022-11-21 at 2 26 52 PM" src="https://user-images.githubusercontent.com/43893061/203143643-6f098f31-f88e-47ae-9264-54a2b0f713b1.png">
 

resolves #249
